api/src/domain/objects/domain.js

```diff
@@ -223,6 +224,12 @@ export const domainType = new GraphQLObjectType({
         return edges
       },
     },
+    claimTags: {
+      description:
+        'List of labelled tags users of an organization have applied to the claimed domain.',
+      type: new GraphQLList(domainTag),
```

Comment:
This can just be a `GraphQLList(GraphQLString)` if handling translation in the loader.